docs/configuring.rst

```diff
@@ -57,6 +64,12 @@ These effect how we display the Table of Contents in the side bar. You can read
 * ``includehidden`` Bool. Specifies if the sidebar includes toctrees marked with the ``:hidden:`` option
 * ``titles_only`` Bool. If True, removes headers within a page from the sidebar.
 
+HTML Context Options
+--------------------
+
+TODO.
```

Comment:
What's the plan for this separately from `Base options`?

I generally try to have TODO free code but if the plan is to quickly put something there I'm ok with this as a stopgap. I would just say remove it and add it completely in another PR. That's probably better than having a TODO.

Reply:
Yes, I'll write a new PR right after this one is merged and before the release.